wgpu/src/image/atlas.rs

```diff
@@ -341,7 +343,7 @@ impl Atlas {
             mip_level_count: 1,
             sample_count: 1,
             dimension: wgpu::TextureDimension::D2,
-            format: wgpu::TextureFormat::Bgra8UnormSrgb,
+            format: self.format,
```

Comment:
I don't think we can change the `format` of the `Atlas` easily like this because we are uploading the image data in BGRA format.

Reply:
I didn't think about that. Just checked with `Rgba8UnormSrgb` which turned ferris blue and thus reverted those changes.